server/entity/Media.ts

```diff
@@ -129,10 +130,10 @@ class Media {
   @UpdateDateColumn()
   public updatedAt: Date;
 
-  @Column({ type: 'datetime', default: () => 'CURRENT_TIMESTAMP' })
+  @DbAwareColumn({ type: 'datetime', default: () => 'now()' })
   public lastSeasonChange: Date;
 
-  @Column({ type: 'datetime', nullable: true })
+  @DbAwareColumn({ type: 'datetime', default: () => 'now()' })
```

Comment:
So this has been implemented just to pass the cypress tests?
It doesn't feel right to modify the codebase just to be compliant with the tests.
Couldn't something else be changed in the cypress tests to fix this issue? If you're struggling to find a solution i can take a deeper look into it.

Reply:
> It doesn't feel right to modify the codebase just to be compliant with the tests.

I agree.

Reply:
It does fix the cypress tests, but I think that's because it also fixes something with typeorm. I didn't see the issue with uses of the @CreationDateColumn but I'd need to make sure that doesn't break existing SQLite or anything and I'm not sure when I'll have time to do that

Reply:
Shockingly I had time today. So postgres doesn't support a column type of datetime. This doesn't seem to be an issue at runtime, but it does cause issues with certain non-runtime uses of the typeorm utilities. In the case of cypress, it's the call to `datasource.synchronize()`.
When using the special date columns (`@CreateDateColumn`), typeorm is able to figure out which type to use dynamically, but it can't for regular columns. So if we still want to play by the rules of typeorm then I believe we need the date field to be dynamic in a way that is like this.
Surprisingly, these 2 fields in `Media` seem to be the only `Dates` that are set manually in the code.
With the latest changes, I was able to run the cypress tests successfully against both sqlite and postgres

Reply:
Thanks, it makes more sense now.
Let me do some more tests and it should be ready to merge.